### PR TITLE
Fix certificate import (#853)

### PR DESCRIPTION
**Pull request by suleymanakbas91**
**Description**

Changes proposed in this pull request:

- Make certificate import work for both alpha and existing way of installation.

### CODE REVIEW

**Review by tobiscr — APPROVED** [review]

File: internal/trust/trust_darwin.go [hunk 1]
```diff
@@ -53,6 +68,11 @@ func (k keychain) StoreCertificate(file string, i Informer) error {
 }
 
 func (keychain) Instructions() string {
+	return "1. Download the certificate: kubectl get secret kyma-gateway-certs -n istio-system -o jsonpath='{.data.cert}' > kyma.crt\n" +
```
Comment (clebs) [inline internal/trust/trust_darwin.go:71]: This step is for the new installation right? In the old installation the cert is still in a configmap in `kyma-system` right?
From the git history, the old instructions were:
`kubectl get configmap net-global-overrides -n kyma-installer -o jsonpath='{.data.global\\.ingress\\.tlsCrt}' | base64 --decode > kyma.crt`

File: internal/trust/trust_windows.go [hunk 2]
```diff
@@ -49,6 +64,13 @@ func (c certutil) StoreCertificate(file string, i Informer) error {
 }
 
 func (certutil) Instructions() string {
+	return "1. Open a terminal window with administrator rights.\n" +
+		"2. Download the certificate: kubectl get secret kyma-gateway-certs -n istio-system -o jsonpath='{.data.cert}' > tmp.txt\n" +
```
Comment (clebs) [inline internal/trust/trust_windows.go:68]: Same issue as above, wrong kubectl command on instructions

File: internal/trust/trust_linux.go [hunk 3]
```diff
@@ -65,6 +80,13 @@ func (c certauth) StoreCertificate(file string, i Informer) error {
 }
 
 func (certauth) Instructions() string {
+	return "1. Download the certificate: kubectl get secret kyma-gateway-certs -n istio-system -o jsonpath='{.data.cert}' > kyma.crt\n" +
```
Comment (clebs) [inline internal/trust/trust_linux.go:83]: Same issue as above, wrong kubectl command on instructions

**Review by clebs — CHANGES_REQUESTED** [review]
Instructions kubectl commands for old installation are incorrect.

**Review by clebs — APPROVED** [review]

**Pull request merged**